src/interpreter/executors/instruction.rs

```diff
@@ -496,5 +490,39 @@ where
     }
 }
 
+/// Computes logarithm for given exponent and base.
+/// Diverges when exp == 0 or base <= 1.
+#[inline(always)] // Force copy of each invocation for optimization, see comments below
+const fn _unchecked_ilog_inner(exp: Word, base: Word) -> u32 {
```

Comment:
Maybe include a reference to where this code was sourced from?

Reply:
Done

Reply:
Not seeing any changes, was a push missed somewhere?

Reply:
Yep, fixed now